ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContextSpec.scala

```diff
@@ -60,11 +60,16 @@ class CommitContextSpec extends AnyWordSpec with Matchers {
     }
 
     "record key as accessed even if it is not available in the input" in {
-      val context = newInstance()
+      val context = newInstance(inputs = Map(aKey -> None))
 
       context.read(aKey) shouldBe None
       context.getAccessedInputKeys shouldBe Set(aKey)
     }
+
+    "throw in case key cannot be found" in {
+      val context = newInstance()
+      assertThrows[Err.MissingInputState](context.read(aKey))
```

Comment:
Please assert here that the key should not be added to the accessed key list.

Reply:
Done.